src/server/xtra1.c

```diff
@@ -152,13 +168,7 @@ static void prt_level(int Ind)
 {
 	player_type *p_ptr = Players[Ind];
 
-	int adv_exp;
-
-	if (p_ptr->lev >= PY_MAX_LEVEL)
-		adv_exp = 0;
-	else adv_exp = (s32b)(player_exp[p_ptr->lev - 1] * p_ptr->expfact / 100L);
-
-	Send_experience(Ind, p_ptr->lev, p_ptr->max_exp, p_ptr->exp, adv_exp);
+	send_indication(Ind, IN_LEVEL, MAX(p_ptr->max_plv, p_ptr->lev), p_ptr->lev);
```

Comment:
This seems unnecessary, as with the similar cases below. Is there not a better way do do this? The giveaway is the fact that the parameters are repeated and the functions appear side-by-side.